app/src/main/res/values/arrays.xml

```diff
@@ -70,6 +72,7 @@
         <item>@style/AppTheme.Dark</item>
         <item>@style/AppTheme.Ford</item>
         <item>@style/AppTheme.BMW</item>
+        <item>@style/AppTheme.MK</item>
```

Comment:
Add an item to the `ThemeList` array. It should be `@style/AppTheme.<your theme name>`. It must be in the same position as the previous two arrays.